libraries/AP_HAL_ChibiOS/hwdef/Pixhawk4/hwdef.dat

```diff
@@ -3,6 +3,14 @@
 
 include ../fmuv5/hwdef.dat
 
+# red and blue LEDs swapped vs fmuv5
+# red LED marked as B/E
+undef PC7
+PC7 LED_RED OUTPUT GPIO(92) HIGH
+# blue LED marked as ACT
+undef PB1
+PB1 LED_BLUE OUTPUT OPENDRAIN GPIO(90)
```

Comment:
This should be GPIO(92), keeping it 90 still ties PB1 to GPIO_A_LED , Pin name is not relevant.